Bio/PDB/PDBList.py

```diff
@@ -665,10 +689,49 @@ def get_seqres_file(self, savefile="pdb_seqres.txt"):
         """Retrieve and save a (big) file containing all the sequences of PDB entries."""
         if self._verbose:
             print("Retrieving sequence file (takes over 110 MB).")
-        url = self.pdb_server + "/pub/pdb/derived_data/pdb_seqres.txt"
+        url = f"{self.pdb_server}/derived_data/pdb_seqres.txt"
         urlretrieve(url, savefile)
 
 
+PDBServer = collections.namedtuple("PDBServer", ["domain", "pdb_directory"])
```

Comment:
Just from a logical perspective, I'd move this to the top of the file. `PDBServer` first, then `PDBList`.

Reply:
And following my comment above about adding some methods to `PDBServer`, maybe indeed a dataclass like you had before would be a better approach, what do you think?